packages/atomic-angular/projects/atomic-angular/project.json

```diff
@@ -1,4 +1,26 @@
 {
   "name": "atomic-angular",
-  "implicitDependencies": ["atomic-angular-builder"]
+  "implicitDependencies": ["atomic-angular-builder"],
+  "targets": {
+    "promote:npm:beta": {
+      "dependsOn": [],
+      "inputs": [],
+      "outputs": [],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "node ../../../../../scripts/deploy/update-npm-tag.mjs beta",
```

Comment:
I think you're going one level too deep

Reply:
![image](https://user-images.githubusercontent.com/12199712/220740473-5b616fb5-dd76-46d8-8e82-c7cd20651f63.png)

Reply:
yeah I was 😅 fixed it and tested it now